cg/cli/workflow/balsamic/base.py

```diff
@@ -173,10 +172,14 @@ def store_housekeeper(context, case_id):
         balsamic_analysis_api.upload_bundle_housekeeper(case_id=case_id)
         LOG.info(f"Storing Analysis in ClinicalDB for {case_id}")
         balsamic_analysis_api.upload_analysis_statusdb(case_id=case_id)
-    except (BalsamicStartError, BundleAlreadyAddedError, FileExistsError) as e:
+    except (BundleAlreadyAddedError, FileExistsError) as e:
         LOG.error(f"Could not store bundle in Housekeeper and StatusDB: {e.message}!")
         balsamic_analysis_api.housekeeper_api.rollback()
         balsamic_analysis_api.store.rollback()
+        balsamic_analysis_api.set_statusdb_action(case_id=case_id, action="hold")
```

Comment:
This action is by "tradition" used for holding the analysis and "Prodteam" will probably be confused if we use it in new ways, we should probably have a discussion on this type of change

Reply:
Will remove for now until a suitable solution introduced in ClinicalDB